src/App.tsx

```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
 
-import Home from './components/Home';
-import Apply from './components/Apply';
-import ApplyDeveloper from './components/ApplyDeveloper';
-import ApplyProductDesigner from './components/ApplyProductDesigner';
-import ApplyBrandDesigner from './components/ApplyBrandDesigner';
-import Partner from './components/Partner';
-import Projects from './components/Projects';
-import Jumpstart from './components/Jumpstart';
-import Events from './components/Events';
-import Board from './components/Board';
-import FAQ from './components/FAQ';
-import NotFound from './components/NotFound';
+import Home from './pages/home/Home';
+import Apply from './pages/apply/Apply';
+import ApplyDeveloper from './pages/apply/ApplyDeveloper';
+import ApplyProductDesigner from './pages/apply/ApplyProductDesigner';
+import ApplyBrandDesigner from './pages/apply/ApplyBrandDesigner';
+import Projects from './pages/projects/Projects';
+import Jumpstart from './pages/jumpstart/Jumpstart';
+import Events from './pages/events/Events';
+import People from './pages/people/People';
+import FAQ from './pages/faq/FAQ';
+import NotFound from './pages/notfound/NotFound';
```

Comment:
thoughts on renaming the files to `index.tsx`? 
That would allow `import Home from './pages/home'`

Reply:
oo interesting, this also exposes a flaw in my directory structure: the ApplyDeveloper/Designer etc pages need to be in their own folder too